CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 
 # PayPal iOS SDK Release Notes
 
+## unreleased
+* PayPalNativePayments
+  * Add deprecated warning in public classes and methods
```

Comment:
Let's update this to make it more similar to the suggestion in the BT SDK: https://github.com/braintree/braintree_ios/pull/1341#discussion_r1649157944

Reply:
addressed here: b7d4b31a22bf715b66da3f8a08023c50d55aadf8